cpu/efr32mg1p/ldscripts/efr32mg1p132f256gm32.ld

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2015-2017 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @addtogroup      cpu_efr32mg1p
+ * @{
+ *
+ * @file
+ * @brief           Memory definitions for the EFR32MG1P132F256GM32 CPU
+ *
+ * @author          Hauke Petersen <hauke.petersen@fu-berlin.de>
+ * @author          Bas Stottelaar <basstottelaar@gmail.com>
+ *
+ * @}
+ */
+
+MEMORY
+{
+    rom (rx)    : ORIGIN = 0x00000000, LENGTH = 262144
```

Comment:
262144 can be written as 256K to make it a little bit more friendly

Reply:
I can change that, but I have another PR coming up that will use the default Cortex-M linker scripts :-)